tests/e2e/test_post_and_cancel_order_live_e2e.py

```diff
@@ -34,24 +34,31 @@
     TokenAmount,
 )
 
-
-E2E_GNOSIS_MAINNET_TESTING_EOA_PRIVATE_KEY = os.getenv(
-    "E2E_GNOSIS_MAINNET_TESTING_EOA_PRIVATE_KEY", ""
-)
-
 COW_TOKEN_GNOSIS_MAINNET_ADDRESS = "0x177127622c4A00F3d409B75571e12cB3c8973d3c"
 WXDAI_GNOSIS_MAINNET_ADDRESS = "0xe91D153E0b41518A2Ce8Dd3D7944Fa863463a97d"
 
-E2E_GNOSIS_MAINNET_TESTING_EOA = Account.from_key(
-    E2E_GNOSIS_MAINNET_TESTING_EOA_PRIVATE_KEY
-)
-E2E_GNOSIS_MAINNET_TESTING_EOA_ADDRESS: Address = E2E_GNOSIS_MAINNET_TESTING_EOA.address
+
+@pytest.fixture
+def e2e_gnosis_mainnet_testing_eoa():
```

Comment:
Converted to a fixture such that the lack of env var doent cause all tests to fail.